### PR TITLE
fix bugs

### PR DESCRIPTION
**Pull request by Glenfiddish**

### CODE REVIEW

File: packages/lb-components/src/components/pointCloudView/hooks/useSingleBox.ts [hunk 1]
```diff
@@ -120,23 +121,40 @@ export const useSingleBox = () => {
 
   /** PointCloud select next/prev one */
   const switchToNextBox = useCallback(
-    (sort = ESortDirection.ascend) => {
+    (sort = ESortDirection.ascend, manual = false) => {
       if (!topViewInstance || selectedIDs.length > 1) {
         return;
       }
 
+      if (pointCloudPattern !== EToolName.Rect && pointCloudPattern !== EToolName.Polygon) {
```
Comment (MMMMoriaty) [inline packages/lb-components/src/components/pointCloudView/hooks/useSingleBox.ts:129]: May be this part of logic doesn't match the function name 'switchToNextBox'

Comment (Glenfiddish) [reply packages/lb-components/src/components/pointCloudView/hooks/useSingleBox.ts:129]: Updated on next commit

**Review by MMMMoriaty — APPROVED** [review]

**Pull request merged**